apps/aehttp/test/aehttp_contracts_SUITE.erl

```diff
@@ -234,11 +234,17 @@ abort_test_contract(Config) ->
     %% Check that we get the abort string back.
 
     AbortString = <<"yogi bear">>,
-    {RevertValue,_} =
-        revert_call_compute_func(Node, APub, APriv, EncodedTestPub,
+    GivenGas = 100000,
+    {RevertValue, Return} =
+        revert_call_compute_func(Node, APub, APriv, EncodedInt3Pub,
```

Comment:
I see this is calling abort_test_int that does not abort directly though in nested call. Ok.